pkg/reconciler/v1alpha1/revision/resources/queue.go

```diff
@@ -92,13 +92,21 @@ func makeQueueContainer(rev *v1alpha1.Revision, loggingConfig *logging.Config, a
 		loggingLevel = ll.String()
 	}
 
+	userGroup := int64(1337)
```

Comment:
It would be better to have these values somewhere we can share with the tests.

Reply:
Done